Procfile

```diff
@@ -1,4 +1,4 @@
-web: make clean && python2.7 manage.py runserver 0.0.0.0:8000
+web: make clean && gunicorn kuma.vagrant-wsgi -b 0.0.0.0:8000
```

Comment:
Good catch. It works the same way as `runserver` for me&mdash;collectstatic only needs to be run when `DEBUG = True`.